skopt/dummy_opt.py

```diff
@@ -4,16 +4,19 @@
 import inspect
 import numbers
 import numpy as np
+from time import time
 
 from collections import Iterable
 from scipy.optimize import OptimizeResult
 from sklearn.utils import check_random_state
 
 from .space import Space
-
+from .utils import create_result
+from .utils import verbose_func
```

Comment:
Should have commented on this before: can we think up a better name? I can't suggest one right now but `verbose_func` seems overly generic and "I won't tell you what I actually do"